README.adoc

```diff
@@ -244,28 +249,30 @@ include::finish/src/main/java/io/openliberty/guides/data/Packages.java[]
 
 [role="code_command hotspot file=0", subs="quotes"]
 ----
-#Update the `Packages.java` class.#
+#Update the `Packages.java` class:#
 `src/main/java/io/openliberty/guides/data/Packages.java`
 ----
 
-Here is an example of a complex query simplified using JDQL.  
+Here is an example of a complex query simplified using JDQL.
 
 ```java
 @Query("WHERE length > :threshold OR height > :threshold OR width > :threshold")
 List<Package> withDimensionLargerThan(float threshold);
 ```
 
-Achieving the same query with Query by Method Name would result in a very long method name and additional parameters. JDQL is a strict subset of the Jakarta Persistence Query Language (JPQL), supporting named and ordinal parameters, as well as a number of operators.
+Achieving the same query with Query by Method Name would result in a very long method name and additional parameters.
+
 
+JDQL supports named and ordinal parameters, as well as a number of operators.
 ```java
 @Query("WHERE length + width + height > ?1")
 List<Package> withTotalDimensionOver(float threshold);
 ```
 
-See the Jakarta Data specification for a more comprehensive overview of the Jakarta Data Query Language.
+See the https://jakarta.ee/specifications/data/1.0/[Jakarta Data specification] for a more comprehensive overview of the Jakarta Data Query Language.
```

Comment:
```suggestion
See the https://jakarta.ee/specifications/data/1.0/jakarta-data-1.0#_jakarta_data_query_language[Jakarta Data specification] for a more comprehensive overview of the Jakarta Data Query Language.
```

This would link to the JDQL section, for the overall spec we could use: https://jakarta.ee/specifications/data/1.0/jakarta-data-1.0